src/qibocal/protocols/resonator_utils.py

```diff
@@ -0,0 +1,360 @@
+import numpy as np
+from numpy.typing import NDArray
+from scipy.ndimage import gaussian_filter1d
+from scipy.optimize import leastsq, newton
+
+
+def get_cable_delay(frequencies: NDArray, z: NDArray, num_points: int = 10) -> float:
+    """
+    Evaluates the cable delay τ caused by the length of the cable and the finite speed of light.
```

Comment:
The first sentence of the docstring should be an extremely concise description of what the function is doing. For sure, it should fit in a single stand-alone line, and we usually place it on the same line of the opening quotes.
```suggestion
    """Evaluates the cable delay :math:`\tau`.

     This delay is caused by the length of the cable and the finite speed of light.
```